radicle-node/src/control.rs

```diff
@@ -131,7 +131,7 @@ fn command<H: Handle<Error = runtime::HandleError>>(
                 }
             }
         }
-        CommandName::TrackedRepos => match handle.tracked_repos() {
+        CommandName::TrackedRepos => match handle.repo_policies() {
```

Comment:
It would make sense to rename the commands too, I would say :)
```suggestion
        CommandName::RepoPolicies => match handle.repo_policies() {
```

Reply:
Oh good catch!